### PR TITLE
More gax paging fixes

### PR DESCRIPTION
**Pull request by tseaver**
Work toward fixing Pubsub system tests using GAX.

Correcting incomplete fixes in #1855.

Note that `Topic.pull` is still problematic (#1869).

### CODE REVIEW

File: gcloud/pubsub/_gax.py [hunk 1]
```diff
@@ -159,14 +165,18 @@ def topic_publish(self, topic_path, messages):
         message_pbs = [_message_pb_from_dict(message)
                        for message in messages]
         try:
-            response = self._gax_api.publish(topic_path, message_pbs)
+            event = self._gax_api.publish(topic_path, message_pbs)
+            if not event.is_set():
+                import pdb; pdb.set_trace()
```
Comment (daspecster) [inline gcloud/pubsub/_gax.py]: Probably don't want this right?

Comment (tseaver) [inline gcloud/pubsub/_gax.py]: D'oh!  Trying to juggle fixes to my own stuff versus debugging `gax-google-python-v1`.

**Comment by tseaver** [comment]
@daspecster I've fixed the lint issues with 2f347b8.  The coverage gap was actually introduced in #1868:  see #1871 for a fix.

**Comment by daspecster** [comment]
Coverage gap? All the CI passed, I think I'm missing your meaning here.

**Comment by tseaver** [comment]
@daspecster I don't know why coveralls shows 100%.  When I run `tox -e cover` locally, it shows a missing branch from line 206 -> exit of `gcloud/test__helpers.py`:  that is what led me to #1871.

/me looks:  I don't think coveralls is doing branch coverage. :(

**Comment by daspecster** [comment]
Hmmm https://github.com/lemurheavy/coveralls-public/issues/31

**Comment by tseaver** [comment]
@daspecster I've rebased against `master` after merging #1871.

**Comment by daspecster** [comment]
Maybe this should be a separate issue, but perhaps we should have travis run the `cover` task separately with the `coveralls` task?

**Comment by daspecster** [comment]
This LGTM @tseaver. Just to confirm, setting `page_size = 0`, the cloud side will just return whatever their max is for page_size? Or should it be `None`?

**Comment by tseaver** [comment]
@daspecster 

> Just to confirm, setting page_size = 0, the cloud side will just return whatever their max is for page_size? Or should it be None?

GAX-generated wrappers use `0` as the default for `page_size`.

**Comment by daspecster** [comment]
Ok, let's merge!

**Comment by tseaver** [comment]
BTW, Travis used to run `cover`:  we dropped it to allow the initial run to complete more quickly (a false economy, maybe).

**Pull request merged**